meson.build

```diff
@@ -52,7 +52,7 @@ libfwupd_lt_current = '2'
 libfwupd_lt_revision = '0'
 libfwupd_lt_age = '0'
 libfwupd_lt_version = '@0@.@1@.@2@'.format(libfwupd_lt_current, libfwupd_lt_age, libfwupd_lt_revision)
-libfwupdplugin_lt_current = '5'
+libfwupdplugin_lt_current = '6'
```

Comment:
need to bump contrib/debian/control.in too (s/libfwupdplugin5/libfwupdplugin6/).

Reply:
Fixed.